Makefile

```diff
@@ -20,17 +20,22 @@ test-server:
 nightwatch:
 	nht nightwatch test/js-success.nightwatch.js
 
-demo:
-	webpack --config webpack.config.test.js --dev
-	$(MAKE) test-server
+test: verify test-unit test-build serve nightwatch
+
+test-dev: verify test-unit-dev
+
+build-demo:
+	webpack --config webpack.config.test.js --dev --watch
 
 deploy: assets
 	node ./_deploy/s3.js
 	$(MAKE) npm-publish
 
-test: verify test-unit test-build test-server nightwatch
+build:
+	export DEV_BUILD=1; webpack
 
-test-dev: verify test-unit-dev
+watch:
+	export DEV_BUILD=1; webpack --watch
```

Comment:
unfortunately not. `make` only understands task names and not bash options so that would throw an error as `--watch` isn't a valid name for a task